platformio.ini

```diff
@@ -30,10 +30,29 @@ lib_deps =
 	https://github.com/OXRS-IO/OXRS-IO-IOHandler-ESP32-LIB
 	https://github.com/SuperHouse/OXRS-SHA-Rack32-ESP32-LIB
 	https://github.com/androbi-com/MqttLogger
+	TFT_eSPI
```

Comment:
Think you need to remove the reference to our custom TFT_eSPI library a few lines up?

Reply:
Yes, I should have mentioned it, but thought this PR would be overruled by your work.
Thanks